lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,132 @@
+# rubocop:disable Style/ClassVars
+
+module Methods
+  def display_guide
+    puts "
+		"
+    puts '1 | 2 | 3'
+    puts '---------'
+    puts '4 | 5 | 6'
+    puts '---------'
+    puts '7 | 8 | 9'
+    puts "
+		"
+  end
+
+  def choose(name0, arr0)
+    puts "its #{name0}'s turn to choose a number from #{arr0}"
+  end
```

Comment:
These two methods `display_guide` and `choose` have `puts`. Please move them inside the `main.rb` file :hammer_and_wrench: